TELCODOCS:764 - Adding known issues for release 1.3.2 of OSC (#54493)
openshift/openshift-docs: sandboxed_containers/sandboxed-containers-1.3-release-notes.adoc

[id="sandboxed-containers-1-3-known-issues"]
== Known issues

* When using {sandboxed-containers-first}, if you set SELinux Multi-Category Security (MCS) labels at the container level, the pod fails to start with the following error:
+
[source,terminal]
----
Error: CreateContainer failed: EACCES: Permission denied: unknown
----
+

> MCS labels set at the container level aren't applied to virtiofsd. The `kata` runtime doesn't have access to the container's security context when creating the VM.

Reviewer: ```suggestion
MCS labels set at the container level are not applied to virtiofsd. The `kata` runtime does not have access to the container's security context when creating the VM.
```

+

> This means that virtiofsd won't run with the appropriate SELinux label and won't be able to access host files on behalf of the containers in the VM; all containers can access all files in the VM.

Reviewer: ```suggestion
This means that virtiofsd won't run with the appropriate SELinux label and cannot access host files on behalf of the containers in the VM; all containers can access all files in the VM.
```

[id="sandboxed-containers-1-3-known-issues"]
== Known issues

* When using {sandboxed-containers-first}, if you set SELinux Multi-Category Security (MCS) labels at the container level, the pod fails to start with the following error:
+
[source,terminal]
----
Error: CreateContainer failed: EACCES: Permission denied: unknown
----
+

> MCS labels set at the container level aren't applied to virtiofsd. The `kata` runtime doesn't have access to the container's security context when creating a sandboxed container.

Reviewer 2: ... when creating the VM.

+

> This means that virtiofsd won't run with the appropriate SELinux label and won't be able to access host files in the VM on behalf of the container. It isn't possible to rely on MCS to isolate files for each container in the VM; all containers can access all files.

Reviewer 2: ... to access host files on behalf of the containers running inside the VM.
